### ISSUE TITLE
User-created public and private Lists to curate user experience as well as help build community

### ISSUE THREAD

**Issue by inkygirl** [bluesky-social/social-app#1056]
**Is your feature request related to a problem? Please describe.**

As Bluesky grows and especially when it goes public, many of us will likely be following more users. The more users we follow, the more our home feeds will be a firehose of posts, difficult to keep up with. The "My Feeds" section helps, but access is unwieldy if we want to quickly browse a custom feed that isn't necessarily the one at the top. This isn't a problem for those who just want to post/announce, but for those of us who actually want to keep up with those we follow and interact with the community, a better solution would be appreciated.

Another challenge: Trying to help promote Bluesky to different communities. I am hoping to get more of the kidlit community moved to Bluesky, for instance, and being able to say "Here's a list of children's book agents I've found so far on Bluesky" along with a public link to Bluesky List of users would help motivate more kidlit creators to move to Bluesky. 

**Describe the solution you'd like**

User-created private and public Lists, similar to what is currently available on Twitter. I'm baffled by why Twitter is the only social media platform thus far with private/public user-curated Lists, so I'm wondering if Lists are difficult to implement from a technical standpoint, but I figure it worth posting here.

With private Lists, a user who only had a few minutes to browse Bluesky could quickly go to a "VIP daily" list (or whatever they name it) and browse. For longer sessions, they could go to categorized Lists to browse, depending on what they're looking for e.g. "Picture Book Illustrators" "Children's Book Editors" "Board Gamers" "VR Enthusiasts" etc.

**Describe alternatives you've considered**

There are currently workarounds, such as manually keeping a separate Google doc or other publicly available list, which some users are doing. Doing this is a pain, though, and also a challenge to keep up longterm. Inevitably these become out-of-date.

For reading experience, I'm using the custom Feeds as a temporary workaround, but it's impractical for quick access as well as for temporary private Lists.

Fingers crossed VERY hard that Bluesky eventually adds community-building tools like Lists, which will make it much easier for us to help promote this platform to specific communities as well as improving individual user experience.

**Additional context**

![Lists-SocialMedia-1000](https://github.com/bluesky-social/social-app/assets/111646238/e9aee4d9-b950-47be-b4ff-73e138ca4ccd)

**Comment by pfrazee** [comment]
Hey hey -- there are no technical blockers to this, it's just a feature we haven't had time to implement yet. It's on the roadmap; we just have a lot of stuff we're getting through. Thanks for the issue (and the fun comic)!

**Comment by PSingletary** [comment]
i thumbs up to thins, but also +1

**Comment by simonetthomas** [comment]
Moreover they could be used (ideally) as inputs to feeds. For what I understand, at the moment the only way to generate a feed from a list of users is to create a "mute list" and unsuscribe from it afterwards, which is not practical at all.

**Comment by seboslaw** [comment]
@pfrazee maybe - as an intermediate solution - we could get non-public mute lists? That way I could create a mute list and then build a custom feed for it. However, I would not want everyone to know who I am including in my list.

**Comment by inkygirl** [comment]
A follow-up, since I notice that no one has been assigned to this feature. Custom feeds are a good temporary solution, but they tend to feature those who post a LOT. Some users lack the time/energy (especially going through a rough time in work and/or personal, struggling to find work etc.), and are only able to post occasionally. These voices tend to be drowned out in a custom feed.

Being able to create my own PRIVATE List would help me curate my own Bluesky experience AND be able to hear those quieter voices. 

Lists are essential for those who follow more than a handful of people and have limited time for social media.

Fingers crossed very very very hard. I know there are SO many things in the pipeline for Bluesky, but I encourage you not to underestimate the value of Lists.

**Comment by pfrazee** [comment]
So we're close to finally getting public lists for curation (backend is working on it now). Private lists are still a bit away.

**Comment by inkygirl** [comment]
Yayyyyy!!!!

**Comment by elius-ren** [comment]
Hi, as Bluesky is now open to the public and new users are bloomed, I believe now is the time to have private list implemented. Is it on schedule yet?
List is getting mandatory as we follow more users since then and Public list is truly doing its job well, but still I think some people like me don't want to disclose the list of my closest people or certain interest to other users.

**Comment by LeGiTiM** [comment]
Can't switch to bluesky until private lists are a thing.
World-wide access to every single interest of a person is excessive in many cases.

The main obstacle I see to this feature is the risk of touchy/illegal topics lists. It can be mitigated with precautionary measures like:
- Private lists are by default personal (owner of the account only)
- Shared access feature is possible, but limited to a small number of other people to prevent wide dissemination of problematic content under the radars (we can share our private lists only to a few of our closest family members, friends or essential coworkers).
- Shared access feature to a medium/large number of people could be unlocked by payment methods (could help Bluesky budget without hurting casual users who will not need this for most of them) that require to expose a physical person as responsible in case of reported illegal downward slides for potential prosecution.

**Comment by mdahmus** [comment]
This is the last thing missing for me which would make Bluesky as good as the other place was, before it wasn't.

**Comment by Go1den** [comment]
I want to add support to this request. I think not having an option to keep lists private / invite only is weird. Public lists are a good way for people to access the same set of users, but what if I don't want people knowing who I put in each list? We're doing myspace top 8 friends all over again.

**Comment by Jonatan-Petler** [comment]
Supporting this - I want to have lists of my favorite posters but I don't want people to know who's on my list, it's mine. The public list system seems great but it needs a private option.

**Comment by pentastich** [comment]
Supporting this. I would like to see private lists. I have multiple interests, and they tend to get mashed together now. For example, right now my feed is being overwhelmed by the US election, while we're in the middle of a nailbiter count here in BC, Canada. And forget about finding my local Victoria interests. Private lists are a REALLY IMPORTANT for people to organize their feeds, and I think they're even more essential as BlueSky grows.

**Comment by kyrioA** [comment]
I am also in favor of a private list feature. I've migrated from Twitter and am very happy with Bluesky's feed curating, and lack of blue checkmarks that respond with assistance by AI. I mainly like to have lists of users for feed organization that I wouldn't necessarily want other people to see on my profile.

**Comment by xor50** [comment]
Yeah, I was just thinking to permanently switch to Bluesky and then noticed... lists are only public?? wtf!
(found this thread via Google when I googled for private lists because I was thinking maybe I missed something. It's impossible something that important is missing... right!?)

**Comment by velocity7** [comment]
Would like to second this. Private lists should be a priority if you want more users to switch to your platform.

**Comment by daffhunt** [comment]
+1 for private lists... would be useful for all sorts of things.

**Comment by troberti** [comment]
Seriously, where is this feature? This prevents me from switching. Lists are the best way to organize users, and private should be the default.

**Comment by julianmorley** [comment]
Bumping this too! When I was looking for this as an existing feature request, I didn't go back more than a year. Didn't realize it's been here since July 2023.

**Comment by nielsle** [comment]
Ideally all lists should  start  out as a private  way to organize your feed into separate feeds. 
 
* Lists should only contain  accounts  that you  follow  (And  you should  count  as a  follower)
* You should be able to hide posts from a  given list in your main  feed  (Similar to Mastodon) 
* Publishing  a list  should be  a  separate action.

This is  the  behavior that I would  expect  from other  sites.

**Comment by monicaellerose** [comment]
> Lists should only contain accounts that you follow (And you should count as a follower)

one part why i disagree with this is - for many journalists use lists to track trends or group sources. so following dozens of extremist accounts wouldn't be an ideal situation here-
similar for following bot accounts that post frequently (like earth quakes near by, or weather reports) - these can be good accounts but add a lot of noise to a typical feed.

**Comment by daffhunt** [comment]
I also disagree with having to follow an account before adding it to a private list. (Fair enough for public ones). Specifically, I would like to keep a private list of people I might decide to follow back later, such as when someone follows me but has no bio or posts to assess them on.

**Comment by AndersRipa** [comment]
Add me to the +1 list

**Comment by Crxssroad** [comment]
Is there anything specifically still blocking this as a feature?

With public lists being a thing already, I can't imagine there's much blocking the ability to restrict lists from view to unauthorized users. Just wondering if there's anything someone with a little extra time on their hands(me😂) can do to submit a PR with the proposed code.

**Comment by jcx616** [comment]
+1 to this idea. Hope its still in the works!

**Comment by JSDopefish** [comment]
I'm also a proponent of private lists - used them forever on Twitter - I can keep track of the schools my kids go to without advertising to the world I'm following them.

**Comment by zaneselvans** [comment]
I am also wanting this feature so that I can easily track collections of users easily without advertising it to the world, which in many applications seems like it can have trust and safety implications.

**Comment by gitcos** [comment]
I very much want private lists, which would be a way for me to create a few personal feeds for myself to check.  Twitter isn't the only one: LiveJournal had these, back in the dawn of subscribe-feed social media almost 25 years ago (and its successor dreamwidth also has them).

In addition to private lists, we need it to be much much easier to manage and curate our lists.  It's tedious and cumbersome now.

**Comment by DigitallyNick** [comment]
I can’t 100% switch to or use Bluesky without implementation of private lists, and private blocking. Everything does not need to be public, can we get some privacy and control on our end please.

**Comment by UrbisR** [comment]
I'm in the same boat as everyone else. This is one feature that I think it pretty important, and if it can't be done in the official app, I'd be interested in a third party client that could create private lists, even if I just had to sync them manually between different devices.

**Comment by rvbgnu** [comment]
+1 for this essential feature!
Content curation, specific industry news reading, following Clients and Partners, etc... All that social network activity is so difficult without Private Lists ! I do understand and support the List being public by default, at protocol level, it is essential too to keep protocol open and interoperable.

The next massive feature for all the apps and platform for #SocialMedia Management, is a Contact manager: imagine having your Contact, followers, and Lists all together in one Private space. It would also solve the problem when some network do not have Private List at all. I'm watching you LinkedIn ;-)

**Comment by ScottKirvan** [comment]
+1 

Following

**Comment by DaRealNim** [comment]
My biggest frustration with Bluesky right now is the fact my following feed is too messy. I've followed a lot of people, some small accounts because I like their art, and some big accounts because I want to be kept up to date on their projects. Since there doesn't seem to be a popularity based ranking system for the posts in the following feed (or at least not one that I've noticed), I often miss posts from the big posters I follow, even though they have way more likes and reposts. I want to be able to separate my follows into categories to easily find all these updates in one place, and how I do that isn't anyone's business but mine. Private lists would be a godsend for this

**Comment by tympanicblock61** [comment]
Following

**Comment by leephillips** [comment]
Private lists are essential for using this type of social media intelligently:

https://lee-phillips.org/howtotwitter/

**Comment by bigjtu** [comment]
I hope the PRIVATE lists are getting closer on the roadmap, as it's the only meaningful feature missing: when implemented, Bluesky finally becomes perfect. ;)

What I'd like to do is to follow all the high signal-to-noise posters separately to focus and to reduce clutter and spam, but I don't want to publicly highlight those posters to all the world. Especially grouping friends together publicly is a privacy issue.

**Comment by tezvii** [comment]
still no private lists. how am I supposed to keep my secret interests secret from the watchful eye of my followers

**Comment by IArnaut2** [comment]
> I hope the PRIVATE lists are getting closer on the roadmap, as it's the only meaningful feature missing: when implemented, Bluesky finally becomes perfect. ;)
> 
> What I'd like to do is to follow all the high signal-to-noise posters separately to focus and to reduce clutter and spam, but I don't want to publicly highlight those posters to all the world. Especially grouping friends together publicly is a privacy issue.

Private lists will come any time now. Maybe in a decade or a century.

**Comment by IArnaut2** [comment]
> Is there anything specifically still blocking this as a feature?
> 
> With public lists being a thing already, I can't imagine there's much blocking the ability to restrict lists from view to unauthorized users. Just wondering if there's anything someone with a little extra time on their hands(me😂) can do to submit a PR with the proposed code.

Bluesky devs are busy making AT Protocol crap that complicates implementing private lists.
